quiz/backend/users/factories.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.hashers import make_password
```

Comment:
This import is not used. Should it stay for generating fake data?

Reply:
Nope just removed it